fontforge/nouiutil.c

```diff
@@ -42,21 +42,29 @@ static void NOUI_IError(const char *format,...) {
     fprintf(stderr, "Internal Error: " );
     vsnprintf(buffer,sizeof(buffer),format,ap);
     str = utf82def_copy(buffer);
-    fprintf(stderr,"%s",str);
-    if ( str[strlen(str)-1]!='\n' )
-	putc('\n',stderr);
-    free(str);
+    if (NULL != str) {
+        fprintf(stderr,"%s",str);
+        if ( str[strlen(str)-1]!='\n' )
+            putc('\n',stderr);
+        free(str);
+    } else {
+        fprintf(stderr, "utf82def_copy failure!\n");
```

Comment:
Throwing the error out because `iconv()` has opinions doesn't seem right to me. Maybe add `fprintf(stderr,"%s",buffer);` here, just to print whatever we have, even if badly encoded?

Reply:
Only issue I could see would be malformed characters being interpreted as terminal control characters

Reply:
@ViliusSutkus89, do you have a sample font + steps to reproduce this?

Reply:
I have a broken iconv that's causing the issue :D But it is still UB. 
The original error message is "The PostScript font name "BGDGBW弫Roboto-Bold" is invalid.\nIt should be printable ASCII,\nmust not contain (){}[]<>%/ or space\nand must be shorter than 63 characters"

Reply:
Broken iconv would mean that any error message would trigger UB here

Reply:
I understand that we use `iconv()` mainly for two things: file names and error messages to the terminal. Maybe we can add separate `iconv_t` descriptors for errors using `ASCII//TRANSLIT` as a fallback? In that case your error message would become "The PostScript font name "BGDGBW?Roboto-Bold...", which is still informative.